drivers/SmartThings/zigbee-lock/src/init.lua

```diff
@@ -34,8 +34,12 @@ local UserStatusEnum            = LockCluster.types.DrlkUserStatus
 local UserTypeEnum              = LockCluster.types.DrlkUserType
 local ProgrammingEventCodeEnum  = LockCluster.types.ProgramEventCode
 
+local socket = require "socket"
```

Comment:
nit: `require "cosock.socket"` instead. It ends up being the same function, but we want to push everyone to use cosock wrapped socket rather than directly accessing the native socket.